test/topology_experimental_raft/test_cdc_generation_data.py

```diff
@@ -0,0 +1,22 @@
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.topology.util import check_token_ring_and_group0_consistency
+
+import pytest
+
+"""
+The injection forces the topology coordinator to send CDC generation data in multiple parts,
+if it didn't the command size would go over commitlog segment size limit making it impossible to commit and apply the command.
+"""
+@pytest.mark.asyncio
+async def test_send_data_in_parts(manager: ManagerClient):
+    config = {
+        'commitlog_segment_size_in_mb': 2
+    }
+
+    first_server = await manager.server_add(config=config)
+
+    async with inject_error(manager.api, first_server.ip_addr, 'cdc_generation_mutations_replication'):
+        await manager.server_add(config=config)
```

Comment:
The test works under the assumption that the error injection properly inflates the size of the CDC generation and grows it beyond the size of the commitlog segment. If that breaks and the inflated size won't be big enough, then the test will stop being useful.

I think you should at least assert that the CDC generation mutation was sent over multiple commands. You could look at the `system.group0_history` table and look at the labels there. The first and the last of the commands that transfer a CDC generation mutation should have a non-empty label already; you could just see whether there is a command of form `"insert CDC generation data (UUID: {}), part 1/{}"` there.

Reply:
Done.